### PR TITLE
Fix example in docs for Update-M365DSCAzureAdApplication

### PR DESCRIPTION
**Pull request by desmay**
#### Pull Request (PR) description
Fix example in docs for Update-M365DSCAzureAdApplication

### CODE REVIEW

**Review by andikrueger — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Change log for Microsoft365DSC
 
+# Unrelease
```
Comment (ricmestre) [inline CHANGELOG.md:3]: should read # UNRELEASED

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+* Fix example in documentation for Update-M365DSCAzureAdApplication
```
Comment (ricmestre) [inline CHANGELOG.md:4]: missing couple of spaces here

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
+
```
Comment (ricmestre) [inline CHANGELOG.md:5]: extra line needs to be removed

File: docs/docs/user-guide/cmdlets/Update-M365DSCAzureAdApplication.md [hunk 4]
```diff
@@ -86,6 +86,6 @@ This function does not generate any output.
 
 -------------------------- EXAMPLE 4 --------------------------
 
-`Update-M365DSCAzureAdApplication -ApplicationName $Microsoft365DSC -Permissions $(Get-M365DSCCompiledPermissionList -ResourceNameList Get-M365DSCAllResources -PermissionType Application -AccessType Read) -Type Certificate -CreateSelfSignedCertificate -AdminConsent -MonthsValid 12 -Credential $creds -CertificatePath c:\Temp\M365DSC.cer`
+`Update-M365DSCAzureAdApplication -ApplicationName 'Microsoft365DSC' -Permissions $(Get-M365DSCCompiledPermissionList -ResourceNameList $(Get-M365DSCAllResources) -PermissionType Application -AccessType Read) -Type Certificate -CreateSelfSignedCertificate -AdminConsent -MonthsValid 12 -Credential $creds -CertificatePath c:\Temp\M365DSC.cer`
```
Comment (ricmestre) [inline docs/docs/user-guide/cmdlets/Update-M365DSCAzureAdApplication.md:90]: great change here LGTM

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
 # Change log for Microsoft365DSC
 
+# UNRELEASED
+ * Fix example in documentation for Update-M365DSCAzureAdApplication
 
 # 1.24.228.1
```
Comment (ricmestre) [inline CHANGELOG.md:6]: nit picking but still missing space :)

Comment (andikrueger) [reply CHANGELOG.md:7]: Always good to have the second pair of eyes 👍

**Pull request merged**